packages/models-library/tests/test_utils_nodes.py

```diff
@@ -39,10 +45,15 @@ def node_id() -> NodeID:
                     "input_bool": True,
                     "input_string": "string",
                     "input_downloadlink": DownloadLink(
-                        downloadLink="http://httpbin.org/image/jpeg"
+                        downloadLink=parse_obj_as(
+                            AnyUrl, "http://httpbin.org/image/jpeg"
+                        )
                     ),
                     "input_simcorelink": SimCoreFileLink(
-                        store=0, path="/path/to/some/file"
+                        store=0,
+                        path=SimcoreS3FileID(
```

Comment:
NOTE that ``SimcoreS3FileID( ... )`` will NOT validate and only have the effect of ``str( ... )``. Validation gets executed when you call ``parse_obj_as(constraint-type, something)```

Reply:
take into account that ``AnyUrl`` is actually also inheriting from ``str`` !

Reply:
yes you are right. I thought that for a test that was ok

Reply:
I mentioned this because you do ``parse_obj_as`` in the same test for ``AnyUrl``